docs/docs/persistent-volumes.md

```diff
@@ -39,69 +43,77 @@ Additionally, you need to set a `residency` in order to tell Marathon to setup a
 }
 ```
 
-## Benefits of using Local Persistent Volumes
-
-- All resources needed to run tasks of your stateful service are dynamically reserved, thus ensuring the ability to relaunch the task on the same node using the same volume when needed.
-- You don't need constraints to pin a task to a particular agent where its data resides
-- You can still use constraints to specify distribution logic
-- Marathon lets you locate and destroy an unused persistent volume if you don't need it anymore
-
-## Prerequisites
+- `containerPath`: The path where your application will read and write data. This can currently be only relative and non-nested (`"data"`, but not `"/data"`, `"/var/data"` or `"var/data"`).
+- `mode`: The mode of the volume. Currently, `"RW"` is the only possible value.
+- `persistent.size`: The size of the persistent volume in MiBs.
 
-In order to be able to create stateful applications using local persistent volumes in Marathon, you need to set 2 command line flags which Marathon will use to reserve/unreserve resources and create/destroy volumes.
+You also need to set the `residency` parameter in order to tell Marathon to setup a stateful application. Currently, the only valid option for this is:
```

Comment:
Not sure "parameter" is the correct here; what's the appropriate term for that part of a JSON app definition?